packages/preview/bamdone-aiaa/0.1.0/template/main.typ

```diff
@@ -0,0 +1,229 @@
+
+#figure(
+  // image("Picture1.png", width:50%),
+  rect(height: 80pt, width: 150pt,[YOUR FIGURE HERE]),
+  caption: [
+    Caption of the figure should go below the figure.
+  ],
+)<a-figure>
+
+Place figure captions below all figures; place table titles above the tables. If your figure has multiple parts, include the labels “a),” “b),” etc. below and to the left of each part, above the figure caption. Please verify that the figures and tables you mention in the text actually exist. Please do not include captions as part of the figures, and do not put captions in separate text boxes linked to the figures. When citing a figure in the text, use the abbreviation “Fig.” except at the beginning of a sentence. Do not abbreviate “Table.” Number each different type of illustration (i.e., figures, tables, images) sequentially with relation to other illustrations of the same type.
+
+Figure axis labels are often a source of confusion. Use words rather than symbols. As in the example to the right, write the quantity “Magnetization” rather than just “M.” Do not enclose units in parenthesis, but rather separate them from the preceding text by commas. Do not label axes only with units. As in Fig. 1, for example, write “Magnetization, kA/m”  not just “kA/m.” Do not label axes with a ratio of quantities and units. For example, write “Temperature, K,” not “Temperature/K.”
+
+Multipliers can be especially confusing. Write “Magnetization, kA/m” or “Magnetization, 103 A/m.” Do not write “Magnetization (A/m) x 1000” because the reader would not then know whether the top axis label in Fig. 1 meant 16000 A/m or 0.016 A/m. Figure labels must be legible, and all text within figures should be uniform in style and size, no smaller than 8-point type.
+
+
+== Equations, Numbers, Symbols, and Abbreviations
+
+$ integral_0^r_2 &F(r,phi) upright(d) r upright(d) phi = [sigma r_2 \/ 2(mu_0)] \ dot&  integral_0^infinity  exp(-lambda | z_j - z_i | ) lambda^-1 J_1 (lambda r_2) J_0 (lambda e_i) upright(d) lambda $ <equation1>
+
+Be sure that the symbols in your equation are defined before the equation appears, or immediately following. Italicize symbols (T might refer to temperature, but T is the unit tesla). Refer to “Eq. (1),” not “(1)” or “equation (1)” except at the beginning of a sentence: “Equation (1) is…” Equations can be labeled other than “Eq.” should they represent inequalities, matrices, or boundary conditions. If what is represented is really more than one equation, the abbreviation “Eqs.” can be used. 
+
+An example of using Equations can be used as follows: It can be referred to as @equation1. Note that this will work when it's an in-sentence equation. However, when a sentence begins with Equation, you will need to use the following to make it work:
+
+#bEquation([@equation1]) when it starts at the beginning of a sentence.
```

Comment:
I recommend kebab-case as the most idiomatic Typst case, but it's not required.

Reply:
It's now `b-equation()` rather than `bEquation` to conform to your suggestion. Thank you.